### ISSUE TITLE
Possibly incorrect example in docs for webpack component relative paths

### ISSUE THREAD

**Issue by cyberwombat** [angular/angular.io#2156]
In the docs https://angular.io/docs/ts/latest/cookbook/component-relative-paths.html#!#webpack it mentions using `require` for styles/components:

````
They can load templates and styles at runtime by setting the component metadata template and style properties with require statements that reference component-relative URLS.
````

However the example given makes no use of require statements:
```
import { Component } from '@angular/core';

import '../../public/css/styles.css';

@Component({
  selector: 'my-app',
  templateUrl: './app.component.html',
  styleUrls: ['./app.component.css']
})
export class AppComponent { }
````
It looks like the example is wrong yes?

**Comment by Foxandxss** [comment]
No, not really. Let me explain.

With Webpack, you **need** to require templates and styles like that description says:

```
templateUrl: require('./my.template.html'),
styleUrls: [require('./some.css')]
```

Now, there is a loader for Webpack that allows you to "ignore" the require like:

```
templateUrl: './app.component.html',
styleUrls: ['./app.component.css']
```

In reality that is doing the same thing as earlier but the loader is doing the `require` for you behind the scenes. We do this because the CLI@Webpack does the same thing as well.

Now, that being said, the text doesn't match what the example says. The problem is that I can write an example for Webpack for the Webpack guide and then you can "borrow" parts of that example in other different guides. So I updated the Webpack example and explanations and since this guide is "borrowing" one snippet to make an example, the explanation is no longer true.

So thanks, I am going to think and see what's the best outcome for this.

**Comment by wardbell** [comment]
Closed by #2157
